lib/tasks/safe_reset.rake

```diff
@@ -10,6 +10,7 @@ namespace :db do
                    .filter { |row| row["schemaname"].include?("public") }
```

Comment:
While we're here, why not just `SELECT * FROM pg_catalog.pg_tables WHERE schemaname = 'public';`?